content/docs/tutorials/fullstack.md

```diff
@@ -35,20 +35,225 @@ In this tutorial, we will be going through the entire process of creating a full
 - Frontend
     - React
     - Bootstrap
-- User Authentication
-    - Auth0
-- Monitoring
-    - [Lambda Powertools](https://awslabs.github.io/aws-lambda-powertools-python/latest/)
-    - Aws Cloudwatch
-- Sending Text and Emails
-    - Twilio
-    - Send Grid
-- Payments
-    - Stripe
-- Unit Testing
-    - Pytest
 
+{{<break 1>}}
+
+## Create Cdev Project
+We will be starting this tutorial from the standard `quick-start` template.
+
+```bash
+cdev init diary-project --template quick-start
+```
+
+Now we can deploy our project to get a live Webhook
```

Comment:
'Webhook' could be change to REST Api because Webhooks are subsets of REST Api but we aren't using this Api as a webhook